oak/common/fuzzer.bzl

```diff
@@ -16,11 +16,11 @@
 
 """Fuzz-related definitions for Oak testing."""
 
-load("@rules_cc//cc:defs.bzl", "cc_binary")
+load("@rules_cc//cc:defs.bzl", "cc_library")
 
 # Inspired by https://github.com/grpc/grpc/blob/618a3f561d4a93f263cca23abad086ed8f4d5e86/test/core/util/grpc_fuzzer.bzl.
 def oak_fuzzer(name, srcs = [], deps = [], **kwargs):
-    cc_binary(
+    cc_library(
```

Comment:
Not sure about this, it does not have a main in our source code, but when compiled with the fuzzer flags it links against a main provided in that library (https://github.com/google/oss-fuzz/blob/6cb657fe6a92ce05df5595da90e17c99ea17a041/projects/oak/build.sh#L60-L61), so effectively the resulting artifact is a binary.

Also I think bazel would not put compiled `cc_library` artifacts under `bazel-bin`, or would it?

Reply:
Is there a way to try it out with OSS-Fuzz?

Reply:
I think these are the relevant instructions: https://google.github.io/oss-fuzz/advanced-topics/reproducing/#reproduce-using-local-source-checkout (but our script expects the output in `bazel-bin`, so if this configuration does not place it there, then I'm pretty sure it will fail).

Reply:
I was hoping not to have to grok the oss-fuzz build system but I guess that's a forlorn hope&hellip;